internal/cloudinit/cloudinit.go

```diff
@@ -96,15 +96,15 @@ swapon /swapfile
 echo "done setting up swap space"
 
 echo "downloading lite engine binary"
-/usr/bin/wget "{{ .LiteEnginePath }}/lite-engine-{{ .Platform.OS }}-{{ .Platform.Arch }}" -O /usr/bin/lite-engine
+/usr/bin/wget --retry-connrefused --retry-on-host-error --retry-on-http-error=503,404,429 --tries=10 --waitretry=1 "{{ .LiteEnginePath }}/lite-engine-{{ .Platform.OS }}-{{ .Platform.Arch }}" -O /usr/bin/lite-engine
```

Comment:
this line --retry-connrefused --retry-on-host-error --retry-on-http-error=503,404,429 --tries=10 --waitretry=1
is being copied 9 times, let's define it as a constant on top and use it.
WGET_RETRY

Reply:
Here the whole string is declared as const, make a variable and concatenate in const string in golang

Reply:
Add waitretry=10